vendor/engines/projects/app/controllers/projects/projects_controller.rb

```diff
@@ -34,20 +36,34 @@ def update
       render action: :edit unless save_project
     end
 
+    def showing_archived?
+      params[:archived] == "true"
+    end
+    helper_method :showing_archived?
+
     private
 
     def project_params
       params.require(:projects_project).permit("active", "description", "name")
     end
 
+    def project_index_params
+      @project.active? ? {} : { archived: "true" }
+    end
+
     def save_project
       if @project.save
         flash[:notice] =
           I18n.t("controllers.projects.projects.#{action_name}.success", project_name: @project.name)
-        redirect_to facility_projects_path(@project.facility)
+        redirect_to facility_projects_path(@project.facility, project_index_params)
```

Comment:
What if we redirected to the show view? That's what I believe we do for Products. I feel like this might be adding unnecessary complexity.